chapters/revisions.tex

```diff
@@ -67,8 +67,8 @@ \subsection{Main changes in Modelica 3.4}\doublelabel{main-changes-in-modelica-3
   Clarified newline, \autoref{strings}. Ticket
   \href{https://github.com/modelica/ModelicaSpecification/issues/1479}{\#1479}.
 \item
-  Allow '"' and define it to be equal to '\textbackslash{}"' (and
-  similarly for '\textbackslash{}?'), \autoref{identifiers}. Ticket
+  Allow '\lstinline!"!' and define it to be equal to '\lstinline!\"!' (and
```

Comment:
Would prefer \lstinline!'"! and similarly for ?

Reply:
To start with, I've changed it to proper pairs of single quotes.  It seems to me that this could be a topic for a separate issue, don't you think?  By leaving it with the single quotes, it should be easy to find all locations later.